app/views/newsletters/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="newsletter-text">
+  <p>
+    Want to be kept up to date with the latest news from Any Rat Rescue? 
+  </p>
+  <button class="subscribe"><%= link_to "Subscribe", "#" %></button>
+  <h1>
```

Comment:
This h1 feels out of place in this `div` perhaps this should be a `header` tag instead and then you could change the div below to a `main` tag